### PR TITLE
HELM-337: exporterNodesWithFlows() query handles criteria differently than other queries

### PR DESCRIPTION
**Pull request by aramosvizcarra**
Allow multiple attribute/value pair filters and values also allow regular expressions
Flows DS
Example: exporterNodesWithFlows(location='$location' & label='.\*com.*')

Also allow multiple attribute/value filters for the nodeFilter function in Entities DS. This one doesn't accept Regex yet since that is done on the backend.
Entity DS now also would work with 
nodeFilterlocation='$location' & label='localhost')

# External References

* JIRA (Issue Tracker): http://issues.opennms.org/browse/HELM-337
* Continuous Integration: [CircleCI](https://circleci.com/gh/OpenNMS/opennms-helm)

### CODE REVIEW

File: docs/modules/datasources/pages/flow_datasource.adoc [hunk 1]
```diff
@@ -57,12 +57,15 @@ exporterNodesWithFlows()
 ```
 
 The filter expression must be an existing node attribute and its corresponding value.
-The value can be a constant or a variable.
+Can accespt more than one filter expression separated by an '&' (<attribute1>=<value1> & <attribute2>=<value2> & ...)
```
Comment (Bonrob2) [inline docs/modules/datasources/pages/flow_datasource.adoc:60]:
```suggestion
Separate multiple filter expressions with an '&' (<attribute1>=<value1> & <attribute2>=<value2> & ...).
```

File: docs/modules/datasources/pages/flow_datasource.adoc [hunk 2]
```diff
@@ -57,12 +57,15 @@ exporterNodesWithFlows()
...
+The value can be a constant, a variable or a regular expression.
```
Comment (Bonrob2) [inline docs/modules/datasources/pages/flow_datasource.adoc:61]:
```suggestion
The value can be a constant, a variable, or a regular expression.
```

**Review by Bonrob2 — CHANGES_REQUESTED** [review]
Minor changes to the documentation.

**Review by Bonrob2 — APPROVED** [review]
Docs look good. Thanks!

File: docs/modules/datasources/pages/flow_datasource.adoc [hunk 3]
```diff
@@ -57,12 +57,15 @@ exporterNodesWithFlows()
...
-The value can be a constant or a variable.
+Separate multiple filter expressions with an '&' (<attribute1>=<value1> & <attribute2>=<value2> & ...).
+The value can be a constant, a variable, or a regular expression.
 
 .Example variable expression
 * exporterNodesWithFlows(<node_attribute>='<value>')
 * exporterNodesWithFlows(location='Default')
 * exporterNodesWithFlows(location='$location')
+* exporterNodesWithFlows(location='$location' & label='localhost')
+* exporterNodesWithFlows(location='$location' & label='.\*com.*')
```
Comment (wakeaney-kr) [inline docs/modules/datasources/pages/flow_datasource.adoc:68]: Is the `\` escape in the right place here? Should it be `\.*com.*` to match a literal `.` ?

Comment (aramosvizcarra) [reply docs/modules/datasources/pages/flow_datasource.adoc:68]: ah is just escaping the formatting for ascii docs only, so it doesn't  considered a bold. Nothing to do with the regex

**Comment by mmahacek** [comment]
I don't have flow data readily available to test.  If @wakeaney-kr is happy with this, I'm happy with this.

**Review by RangerRick — APPROVED** [review]

**Pull request merged**